### ISSUE TITLE
tooltip reads F4 to knit but code says CTRL+e

### ISSUE THREAD

**Issue by timelyportfolio** [ramnathv/rNotebook#4]
tooltip reads F4 to knit but code says CTRL+e

Also, on Windows in Chrome, CTRL+e does not work.  Rather it goes to the address bar and says "enter query".

**Comment by ramnathv** [comment]
Yes. I havent spent time on the tooltips yet. Thanks for bringing it to my attention.

If you have suggestions on better keyboard shortcuts, please post here.
